### ISSUE TITLE
Add option to move call notification pop-up in the middle of the screen.

### ISSUE THREAD

**Issue by WPFilmmaker** [BelledonneCommunications/linphone-desktop#206]
As a poweruser I like where the popups stays now. However for less techsavvy users it would be great having the popup for an incoming call in the middle of the screen, like skype does.

**Comment by zahraesm** [comment]
have you been able to build this project for windows 10?

**Comment by WPFilmmaker** [comment]
@zahraesm no, but if you have issues I recommend opening a ticket here on github or try the dev mailinglist https://lists.nongnu.org/mailman/listinfo/linphone-developers
